xml/obs/systemsmanagement:Agama:Devel.xml

```diff
@@ -3,8 +3,9 @@
     dist_path="obspublish-other::openqa/systemsmanagement:Agama:Devel/"
     distri="opensuse"
     version="agama-9.0"
-    archs="x86_64 aarch64 ppc64le s390x">
+    archs="x86_64 aarch64 ppc64le">
     <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/">
         <hdd filemask="agama-installer.*-openSUSE-Build.*iso$"/>
     </flavor>
+    <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/" archs='s390x' iso="extract_as_repo"/>
```

Comment:
hm, I think when adding `<flavor>` with the same name it just overwrites the previous one. This is why x86_64 iso  is not in print_rsync_iso output anymore